fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/UserPermission.java

```diff
@@ -86,13 +87,15 @@ public View getView() {
   }
 
   @Data
+  @EqualsAndHashCode(callSuper = false)
```

Comment:
Do we know what gets generated here for the `Set`s below? For lists, equality is based on a simple address check - is the same true for sets? If equals between sets is handled right, this is fine.